.cbmc-batch/source/openssl/evp_override.c

```diff
@@ -225,6 +495,19 @@ const EVP_MD *EVP_sha512() {
     return &md;
 }
 
+/* Description: Return the size of the message digest when passed an EVP_MD or an EVP_MD_CTX structure, i.e. the size of
+ * the hash.
+ */
+int EVP_MD_size(const EVP_MD *md) {
+    if (md->from == EVP_SHA256) {
+        return 256;
+    }
+    if (md->from == EVP_SHA384) {
+        return 384;
+    }
+    return 512;
```

Comment:
Is this the only other possibility. It might be safer to `assert(0)` if its not `SHA512'